apidump/apidump.go

```diff
@@ -245,19 +246,27 @@ func compileRegexps(filters []string, name string) ([]*regexp.Regexp, error) {
 func Run(args Args) error {
 	args.lint()
 
+	debugEnabled := viper.GetBool("debug")
+
+	if args.EnableOutbound {
+		printer.Warningln("Enabling collection of outbound traffic. The \"--enable-outbound-collection\" flag is deprecated and may be removed in a future release.")
+	}
+
 	// Get the interfaces to listen on.
 	interfaces, err := getEligibleInterfaces(args.Interfaces)
 	if err != nil {
 		return errors.Wrap(err, "failed to list network interfaces")
 	}
 
 	// Build inbound and outbound filters for each interface.
-	inboundFilters, outboundFilters, err := createBPFFilters(interfaces, args.Filter, 0)
+	inboundFilters, outboundFilters, err := createBPFFilters(interfaces, args.Filter, args.EnableOutbound || debugEnabled, 0)
 	if err != nil {
 		return err
 	}
 	printer.Debugln("Inbound BPF filters:", inboundFilters)
-	printer.Debugln("Outbound BPF filters:", outboundFilters)
+	if args.EnableOutbound {
```

Comment:
args.EnabledOutbound || debugEnabled?  Or if len(outboundFilters) > 0 to cover both those cases? Or maybe just make it unconditional?

Reply:
Changed to `if capturingOutbound` in 5e0a89a.